test/cmdlineTests.sh

```diff
@@ -28,27 +28,86 @@
 
 set -e
 
-REPO_ROOT="$(dirname "$0")"/..
+REPO_ROOT=$(cd "$REPO_ROOT" && pwd)
```

Comment:
I mean it references `REPO_ROOT` which is just an empty string at that point, so effectively it does `cd` and doesn't changes the current directory.